packages/vscode-extension/src/webview/components/UrlSelect.tsx

```diff
@@ -117,8 +117,8 @@ function UrlSelect({ onValueChange, navigationHistory, disabled, dropdownOnly }:
   };
 
   useEffect(() => {
-    setInputValue(navigationHistory[0]?.displayName ?? "");
-  }, [navigationHistory[0]?.id]);
+    setInputValue(navigationHistory[0]?.displayName ?? "/");
+  }, [navigationHistory]);
```

Comment:
why this change? the navigationHistory bit will change with every single state update (even unrelated to navigation) as we clone the whole project state with every change